sans/reduction.ipynb

```diff
@@ -261,10 +302,13 @@
     "        dims=['Q'],\n",
     "        unit=sc.units.one/sc.units.angstrom,\n",
     "        values=np.geomspace(0.008, 0.6, num=55))\n",
-    "    # TODO QResolution\n",
+    "   \n",
     "    d = sc.Dataset({'data':data, 'norm':solid_angle(data)*direct_beam})\n",
+    "    wav_edges = d.coords['wavelength'].copy()\n",
     "    to_bin_centers(d, 'wavelength')\n",
     "    d = sc.neutron.convert(d, 'wavelength', 'Q', out=d) # TODO no gravity yet\n",
+    "    dq = q_resolution(wav_edges, moderator, d)\n",
```

Comment:
No need for copy of `wav_edges`, just move this up by two lines?

Reply:
Yes. I had originally thought I could use the q calculated and set on d as part of unit conversion, but I cannot, so yes copy can now be avoided.